debian/droidian-quirks-device.droidian-boot-wlan.service

```diff
@@ -1,14 +1,17 @@
 [Unit]
-Description=Enables WLAN on devices with boot_wlan
+Description=Enables WLAN on boot
 Before=lxc@android.service
```

Comment:
I wonder if it would be worth running this *after* the halium container runs. For kumano/bahamut I'm already doing that for /dev/wlan ---> https://github.com/droidian-devices/adaptation-sony-kumano/blob/bookworm/debian/adaptation-sony-kumano-configs.droidian-kumano-wlan-enable.service